### DISCUSSION TITLE
Zed

### DISCUSSION THREAD

**Discussion by sgoudham** [catppuccin/catppuccin#1826] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Possibly, when support is available

### What is Zed?
You can find out about Zed [here](https://zed.dev/)!

### What's blocking us?
Currently theming is being reworked by the lovely people at Zed, which is being tracked here:
- https://github.com/zed-industries/community/discussions/1780

Once themes are user-editable, we should look to port Catppuccin to it :D

**Comment by Zazucki** [comment]
https://zed.dev/blog/user-themes-now-in-preview

**Comment by tecandrew** [comment]
starting working on a port for Zed using their vscode->Zed theme importer tool! there are still some differences, but it's got the majority of the coloring there 😮‍💨

the terminal colors need to be adjusted. finding that there's some colors that conflict with the termina text making it hard to read

### Latte

<img width="1552" alt="Screenshot 2024-02-02 at 6 46 49 PM" src="https://github.com/catppuccin/catppuccin/assets/21319730/19ffd80a-f412-4460-8ad4-3fe80f286a18">


### Mocha

<img width="1552" alt="Screenshot 2024-02-02 at 6 47 11 PM" src="https://github.com/catppuccin/catppuccin/assets/21319730/922d84fd-5139-4754-9a49-90f44ad8f120">

### Frappe
<img width="1552" alt="Screenshot 2024-02-02 at 6 49 01 PM" src="https://github.com/catppuccin/catppuccin/assets/21319730/88865205-896f-4888-8e27-0d1a10fab6b4">


### Macchiato

<img width="1552" alt="Screenshot 2024-02-02 at 6 49 15 PM" src="https://github.com/catppuccin/catppuccin/assets/21319730/0cbff86d-f1b3-45f0-94d3-01b9534614cd">

**Comment by tecandrew** [comment]
found the Catppuccin port creation doc and made a public repo for an official port!

https://github.com/tecandrew/zed

edit:
fixed a lot of the poor contrast coloring making it hard to read text and buttons. i used the https://github.com/catppuccin/catppuccin/blob/main/docs/translation-table.md with some manual adjustments to make it more consistent across all panes in the editor
![image](https://github.com/catppuccin/catppuccin/assets/21319730/7aad9752-b5e5-45e8-ba4a-aab410b9c555)

**Comment by sgoudham** [comment]
Closing as further discussion should take place in #2262
